package.json

```diff
@@ -52,9 +52,8 @@
     "base64url-universal": "^2.0.0",
     "chai": "^4.3.7",
     "credentials-context": "^2.0.0",
-    "data-integrity-test-suite-assertion": "github:w3c-ccg/data-integrity-test-suite-assertion#make-vc-gen-suite-configurable",
+    "data-integrity-test-suite-assertion": "github:w3c-ccg/data-integrity-test-suite-assertion#add-suite-verifier-tests",
```

Comment:
does this belong in this pr?

Reply:
it snuck in, but this is actually the branch we should be on as make-vc-gen-suite-configurable https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/39 was merged this week.